router/router.go

```diff
@@ -165,11 +162,7 @@ func New(cfg Config) http.Handler {
 
 	babbageRouter := router.PathPrefix("/").Subrouter()
 	babbageRouter.Use(allRoutesMiddleware)
-	if cfg.LegacyCacheProxyEnabled {
-		babbageRouter.PathPrefix("/").Handler(cfg.ProxyHandler)
-	} else {
-		babbageRouter.PathPrefix("/").Handler(cfg.BabbageHandler)
-	}
+	babbageRouter.PathPrefix("/").Handler(handler)
```

Comment:
this if statement still needs to be in there to catch other traffic.

Reply:
spoke about this